lib/srv/authhandlers.go

```diff
@@ -65,6 +81,34 @@ type AuthHandlers struct {
 	Clock clockwork.Clock
 }
 
+// NewAuthHandlers initializes authorization and authentication handlers
+func NewAuthHandlers(
+	server Server,
+	component string,
+	accessPoint auth.AccessPoint,
+	fips bool,
+	emitter events.Emitter,
+	clock clockwork.Clock,
+) (*AuthHandlers, error) {
```

Comment:
If you're making a constructor method, let's then also make a respective `AuthHandlersConfig` struct so that we don't have to pass 6 arguments.